pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[project]
+name = "A2rchi"
+version = "0.1.0"
+description = "An AI Augmented Research Chat Intelligence for MIT's subMIT project in the physics department"
```

Comment:
With Christoph pushing for this project to be more and more general, not sure if we want to specifically include subMTI in the description here.

Reply:
Sounds good, I can reword this!